### ISSUE TITLE
Code Duplication

### ISSUE THREAD

**Issue by darkvergus** [AlanMorel/MapleServer2#479]
According to the report attached theres about 7% of code duplicated.

[report.zip](https://github.com/AlanMorel/MapleServer2/files/7531118/report.zip)

Example:
![image](https://user-images.githubusercontent.com/1348762/141598584-6052b6f2-8e20-4773-b494-7a8f4167b1cf.png)

If we go to the lines we will see this.

![image](https://user-images.githubusercontent.com/1348762/141598592-3035c388-af72-49b9-970e-b93f75a5cc4d.png)

![image](https://user-images.githubusercontent.com/1348762/141598601-3e107708-0a6e-4497-9610-4e73f5e3d533.png)

Basically we need to avoid this
